src/network/sbfUdpMulticast.c

```diff
@@ -60,15 +60,20 @@ sbfUdpMulticast_create (sbfUdpMulticastType type,
         if (bind (s->mSocket, (struct sockaddr*)&sin, sizeof sin) < 0)
             goto fail;
 
-        memset (&im, 0, sizeof im);
-        im.imr_multiaddr.s_addr = address->sin_addr.s_addr;
-        im.imr_interface.s_addr = interf;
-        if (setsockopt (s->mSocket,
-                        IPPROTO_IP,
-                        IP_ADD_MEMBERSHIP,
-                        (void*)&im,
-                        sizeof im) < 0)
-            goto fail;
+        unsigned char *ip = (unsigned char *)&address->sin_addr.s_addr;
+        int i = ip[0] & 0xFF;
+        if(i >= 224)
+        {
```

Comment:
You should be able to merge those 3 lines into something like this 

if ((address->sin_addr.s_addr & 0xff) > 244)